Week3/homework/project.js

```diff
@@ -0,0 +1,50 @@
+
+"use strict";
+
+function cardValidator(creditCardNumber){
+    let msg = "";
+    const cardNumStr = creditCardNumber.toString();
+    let sum = 0;
+    for(let i = 0; i < cardNumStr.length; i++){
+       sum += parseInt(cardNumStr[i]);
+    }
+    function allTheSame(cardNumStr) {
+        return cardNumStr.split('').every(char => char === cardNumStr[0]);
+    }
```

Comment:
👏 for the logic. However, to find out if we have atleast two unique numbers, I would like to introduce to you a new addition in JavaScript called `Sets`. A **set** can only contain unique entries. Thus, instead of using the `filter` method here, what you could also do is:

```
const uniqueLength = new Set(Array.from(`${cardNum}`).size; // uniqueLength will be 1 if there are no unique characters and will be more than 1 if there are atleast two. You can use this instead of a filter method in such a case 🙂 
```